src/omerocrate/uploader.py

```diff
@@ -225,6 +224,27 @@ def find_images(self) -> Iterable[tuple[Identifier, Path]]:
             file_path = result['file_path']
             yield file_path, Path(urlparse(file_path).path)
 
+    def find_existing_images(self) -> Iterable[tuple[Identifier, int]]:
+        """
+        Finds images with existing OMERO image IDs that should be processed but not uploaded to
+        OMERO (imageID is specified and upload value is false). This is typically used for adding
+        segmentations to existing images.
+        Can be overridden to customize the query.
+
+        Returns:
+            Yields tuples of (image URI, image ID).
+        """
+        for result in self.select_many("""
```

Comment:
Just noting that people who override `find_images` will also have to override `find_existing_images` to make them find the same things. There is an argument for making this function take the full set of images as an argument, and returning the subset of them that already exists in OMERO.